src/Command/CopyCronTasksCommand.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Acquia\Cli\Command;
+
+use AcquiaCloudApi\Endpoints\Crons;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class CopyCronTasksCommand.
+ */
+class CopyCronTasksCommand extends CommandBase {
+
+  protected static $defaultName = 'copy:cron';
```

Comment:
Let's put this in an existing namespace. How about app:copy-cron ?

Reply:
When you check API commands like below, you'd even vote for `app:cron-copy` instead.

```
  api:environments:cron-create                         Creates a scheduled job with cron.
  api:environments:cron-delete                         Deletes a cron job.
  api:environments:cron-disable                        Disables a cron job.
  api:environments:cron-enable                         Enables a cron job.
  api:environments:cron-find                           Return details about a specific cron job.
  api:environments:cron-job-list                       Return environment cron jobs.
  api:environments:cron-update                         Modify an existing scheduled job.
```

Reply:
This I tried to check and tried to add in the `api:environments:*` namespace but couldn't as I couldn't. Seems api commands are created from `acquia-spec.yaml` and can be used only where we have cloud api available straight. In

Reply:
Yeah don't add it in the `api` namespace, add it in the `app` namespace instead.